apps/laboratory/src/pages/library/ethers-no-email.tsx

```diff
@@ -23,6 +23,7 @@ const modal = createAppKit({
   },
   termsConditionsUrl: 'https://reown.com/terms-of-service',
   privacyPolicyUrl: 'https://reown.com/privacy-policy',
+  enableLegalCheckbox: true,
```

Comment:
I think we could move this flag to `options.features` object. Like `features.legalCheckbox`? We want to collect all features under this flag